### PR TITLE
CSC Halo filter: adding classes for nex filter

### PR DESCRIPTION
**Pull request by lathomas**
So far, the results of the two new working points of the filter were only stored as methods in CSCHaloData. 
I have now created two EDFilters for each of them and added them to the list of metFilters. 
I also added them to the lists of flags in MINIAOD. 
Let me know if there's any problem ! 

Thanks a lot, 

Laurent

### CODE REVIEW

File: RecoMET/METFilters/plugins/CSCTightHalo2015Filter.cc [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDFilter.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DataFormats/METReco/interface/BeamHaloSummary.h"
+
+class CSCTightHalo2015Filter : public edm::EDFilter {
```
Comment (slava77) [inline RecoMET/METFilters/plugins/CSCTightHalo2015Filter.cc]: this should be a edm::stream or edm::global (the latter fits better)

Comment (lathomas) [inline RecoMET/METFilters/plugins/CSCTightHalo2015Filter.cc]: Done. I have updated the two new classes and the original one as well.

**Comment by lathomas** [comment]
@slava77  If you are happy with the changes, can you trigger the tests? 
Thanks a lot !

**Comment by slava77** [comment]
@cmsbuild please test

@lathomas 
this will need to be signed by AT
I have just made some quick suggestions to speed up the process

**Comment by monttj** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**